sechub-shared-kernel/src/main/java/com/daimler/sechub/sharedkernel/mapping/MappingIdentifier.java

```diff
@@ -31,75 +32,91 @@ private MappingIdentifier(String id, MappingType type) {
         requireNonNull(type);
         assertValidId(id);
         assertNoDuplicate(id);
-        
+
         this.id = id;
-        this.type=type;
+        this.type = type;
         StaticRef.map.put(id, this);
     }
 
     static void assertNoDuplicate(String id) {
         MappingIdentifier identifierFound = getIdentifierOrNull(id);
-        if (identifierFound!=null) {
-            throw new IllegalStateException("Implementation failure, duplicate detected! Mapping identifier:"+identifierFound+" has already id:"+id);
+        if (identifierFound != null) {
+            throw new IllegalStateException("Implementation failure, duplicate detected! Mapping identifier:" + identifierFound + " has already id:" + id);
         }
     }
 
     static void assertValidId(String id) {
         ValidationResult result = StaticRef.mappingIdValidation.validate(id);
         /* ensure enumeration ids are always valid */
         if (!result.isValid()) {
-            throw new IllegalStateException("Implementation failure, mapping id not valid:"+result.getErrorDescription());
+            throw new IllegalStateException("Implementation failure, mapping id not valid:" + result.getErrorDescription());
         }
     }
 
     public String getId() {
         return id;
     }
-    
+
     public MappingType getType() {
         return type;
     }
-    
-    
+
     /**
      * Get identifier by given id
+     * 
      * @param id
      * @return identifier or <code>null</code> when not found
      */
     public static MappingIdentifier getIdentifierOrNull(String id) {
-        if (id==null) {
+        if (id == null) {
             return null;
         }
         return StaticRef.map.get(id);
     }
-    
-    public static enum MappingType{
-        GLOBAL_CONFIGURATION,
-        ADAPTER_CONFIGURATION
+
+    /**
+     * We got different types for mappings. This is to identify a mapping stands for
+     * @author Albert Tregnaghi
+     *
+     */
+    public static enum MappingType {
+        /* global common configuration mapping */
+        COMMON_CONFIGURATION,
+        
+        /* a configuration mappping for adapters */
+        ADAPTER_CONFIGURATION,
+
+        /* mapping for product executor configuration parameters */
+        PRODUCT_EXEC_CFG_PARAM
```

Comment:
Please rename this to `PRODUCT_EXECUTOR_CONFIG_PARAMS`. See comment in line 22.

Reply:
Renamed to `PRODUCT_EXECUTOR_CONFIGURATION_PARAMETER`